src/models/User.js

```diff
@@ -5,6 +5,10 @@ const UserSchema = mongoose.Schema({
     type: String,
     required: true,
   },
+  vatsimCID: {
+    type: Number,
+    required: true,
```

Comment:
Is this definitely a required field?

Reply:
Yes because user can't use the system without being a VATSIM CID, would be impossible to determine their location without their CID and the VATSIM API.